modules/drouting/drouting.c

```diff
@@ -537,6 +553,10 @@ static void dr_raise_event(pgw_t *gw)
 {
 	evi_params_p list = NULL;
 	str *txt;
+	if ((gw_status_replicate_cluster > 0) && gw)
```

Comment:
You should not trigger the replication from this event function - please see my comment on the replicate_gw_status_update() function, where you should trigger the event if the incoming replication changed the gw status. And in the current format this will lead to a replication storm :). Maybe dr_raise_event() should take one more flag (like do_replication) ; and when called from replicate_gw_status_update() to set it on FALSE.